product_packaging_level/models/product_packaging.py

```diff
@@ -8,8 +8,8 @@
 
 class ProductPackaging(models.Model):
     _inherit = "product.packaging"
-    _order = "product_id, level_sequence"
 
+    sequence = fields.Integer(compute="_compute_sequence", store=True)
```

Comment:
```suggestion
    sequence = fields.Integer(compute="_compute_sequence", store=True, readonly=False)
```
`readonly=False` is need otherwise is not possible to edit the field.